ckanext/dataset_subscriptions/actions/whatsapp_notifications.py

```diff
@@ -96,3 +110,77 @@ def _validate_plugin_extras(extras):
     for field in CUSTOM_FIELDS:
         out_dict[field['name']] = extras.get(field['name'], field['default'])
     return out_dict
+
+
+def get_phonenumber(user_dict):
+    if "activity_streams_whatsapp_notifications" in user_dict and "phonenumber" in user_dict:
+        if user_dict["activity_streams_whatsapp_notifications"] and user_dict["phonenumber"]:
+            phonenumber = user_dict["phonenumber"]
+            return phonenumber
+    return False
```

Comment:
That's a sidenote to the comment above but it's better to return None in this case so we're returning a single optional type from a function instead of bool/string.